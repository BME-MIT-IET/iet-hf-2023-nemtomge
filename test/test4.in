loadmap map.json
start V1 
set Scientist V1 PL1 
give Active V1 Stun 5 
show Active V1 
show Map 
move V1 PL2 
show Map
exit
