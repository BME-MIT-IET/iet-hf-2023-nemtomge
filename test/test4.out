Map from file map.json loaded
The game has been started. Players are: V1 
Scientist V1 set on field PL1 
Scientist V1 has been given active agent Stun time: 5 
V1's active effects are:
Stun(5) 
The map consists of these non-empty fields: 
Plain(PL1): 
 - V1 
Scientist V1 couldn't move
The map consists of these non-empty fields: 
Plain(PL1): 
 - V1
